server/odc-service/src/main/java/com/oceanbase/odc/service/connection/database/DatabaseService.java

```diff
@@ -793,6 +798,14 @@ public void updateObjectLastSyncTimeAndStatus(@NotNull Long databaseId,
         databaseRepository.setObjectLastSyncTimeAndStatusById(databaseId, new Date(), status);
     }
 
+    @SkipAuthorize("odc internal usage")
+    @Transactional(rollbackFor = Exception.class)
+    public void refreshExpiredPendingDBObjectStatus() {
+        this.databaseRepository.setObjectSyncStatusByObjectSyncStatusAndObjectLastSyncTimeBefore(
```

Comment:
we may add log here ,how much rows affected.

Reply:
done